tests/foreman/cli/test_repository.py

```diff
@@ -1937,7 +1937,7 @@ def test_positive_srpm_list_end_to_end(self):
         # Remove uploaded SRPM
         Repository.remove_content({
             'id': new_repo['id'],
-            'ids': [Srpm.list()[0]['id']],
+            'ids': [Srpm.list({'repository-id': new_repo['id']})[0]['id']],
```

Comment:
what is the difference between id and id's? is that both required while removing the content?

If I'm trying to understand here if I removed the repository it does not deletes the all associated Srpms, I could be wrong?

Reply:
Ids => We could pass multiple SRPM packages IDs under it .
id=> id filtering the first package ID only in that repo.

Reply:
means we can have the privilege to delete specific srpms from the uploaded repo?

Reply:
yes, this is how it works for every content i.r. rpm/file/ ..etc

Reply:
not for module streams :)

Reply:
I mean content which can upload same delete as well :) ... I am not sure if module streams can upload manual :)